tsconfig-node.json

```diff
@@ -31,7 +31,8 @@
       "urijs",
       "styled-components" // eventually it will be required anyway for SSR.
       //"react"
-    ]
+    ],
+    "skipLibCheck": true
```

Comment:
As mentioned [here](https://github.com/TerriaJS/terriajs/issues/5219#issuecomment-1317799015), this is required to fix compatibility between the latest version of mobx-react that we can easily use, and TS 4.8+. As I understand it, TS still type checks libraries we actually _use_, this flag just makes it skip type-checking _everything_. So despite the scary name, I don't think this is a bad flag to use.